composer.json

```diff
@@ -19,7 +19,7 @@
         "jackalope/jackalope-transport": "*",
         "php": ">=5.3.3",
         "ext-xml": "*",
-        "phpcr/phpcr-utils": "~1.1",
+        "phpcr/phpcr-utils": "~1.3",
```

Comment:
if we consider https://github.com/phpcr/phpcr-utils/pull/153 a bugfix, this could be `~1.2, >=1.2.6`